pywinauto/controls/win_base_wrapper.py

```diff
@@ -389,6 +391,8 @@ def capture_as_image(self, rect=None):
             return None
 
         if rect:
+            if not isinstance(rect, RECT):
+                raise TypeError("Unexpected type: {} instead {}".format(type(rect), type(RECT())))
```

Comment:
I'd suggest `"capture_as_image() takes rect of type {} while incorrect type {} is given".format(RECT, type(rect))`.